docs/pages/api-reference/liveblocks-emails.mdx

```diff
@@ -6,111 +6,840 @@ meta:
+            {emailData.comments.map((comment) => (
+              <div key={comment.id}>
+                <div>
+                  @{comment.author.id} at {comment.createdAt}
+                </div>
+                <div>{comment.reactBody}</div>
+              </div>
+            ))}
+          </div>
+        );
+        break;
+      }
+    }
+
+    // Send your email
+    // ...
+  }
+  // +++
+
+  return new Response(null, { status: 200 });
 }
 ```
 
-### prepareThreadNotificationAsHtml [#prepare-thread-notification-as-html]
+<PropertiesList title="Returns">
+  <PropertiesListItem
+    name="value"
+    type="ThreadNotificationEmailDataAsReact | null"
+  >
+    Returns comment information, and a formatted React body, ready for use in emails. Returns `null` if there are no unread mentions or comments. The result has two formats depending on whether this notification is for a *single unread mention*, or for *multiple unread replies*:
+
+    ```js title="Unread mention" isCollapsable isCollapsed
+    {
+      type: "unreadMention",
+      roomId: "my-room-id",
+
+      // An unread mention has just one comment
+      comment: {
+        id: "cm_asfs8f...",
+        threadId: "th_sj30as..."
+        createdAt: Date <Fri Dec 15 2023 14:15:22 GMT+0000 (Greenwich Mean Time)>,
+
+        // The formatted comment, pass it to React `children`
+        reactBody: { /* ... */ },
+
+        author: {
+          id: "aurelien@example.com",
+          info: { /* Custom user info you have resolved */ },
+        },
+      },
+    }
+    ```
+
+    <div className="-mt-2">
+
+    ```js title="Unread replies" isCollapsable isCollapsed
+    {
+      type: "unreadReplies",
+      roomId: "my-room-id",
+
+      // Unread replies means multiple comments
+      comments: [
+        {
+          id: "cm_asfs8f...",
+          threadId: "th_sj30as..."
+          createdAt: Date <Fri Dec 15 2023 14:15:22 GMT+0000 (Greenwich Mean Time)>,
+
+          // The formatted comment, pass it to React `children`
+          reactBody: { /* ... */ },
+
+          author: {
+            id: "aurelien@example.com",
+            info: { /* Custom user info you have resolved */ },
+          },
+        },
+
+        // More comments
+        //...
+      ],
+    }
+    ```
+
+    </div>
 
-Prepares data from a
-[`ThreadNotificationEvent`](/docs/platform/webhooks#Thread-notification) and
-convert comment bodies as an html safe string.
+  </PropertiesListItem>
+</PropertiesList>
 
-It returns a `ThreadNotificationEmailDataAsHtml` or `null` if there are no
-unread comments.
+<PropertiesList title="Arguments">
+  <PropertiesListItem name="client" type="Liveblocks" required>
+    A [`Liveblocks`](/docs/api-reference/liveblocks-node#Liveblocks-client)
+    Node.js client.
+  </PropertiesListItem>
+  <PropertiesListItem name="event" type="ThreadNotificationEvent" required>
+    An event from a
+    [`ThreadNotificationWebhook`](/docs/platform/webhooks#Thread-notification).
```

Comment:
I'm struggling a bit for a developer standpoint what a `ThreadNotificationWebhook` is 🤔 
Did you wanted to mean: "An event from a from webhook"?

Reply:
I see what you mean! I've made it more clear, let me know if it still isn't good enough.

Reply:
Super clear 👍🏻  Thanks 🙏🏻